freight_bringup/config/analyzers.yaml

```diff
@@ -0,0 +1,67 @@
+pub_rate: 1.0
+base_path: '' # Optional, prepended to all diagnostic output
+analyzers:
```

Comment:
Please also aggregate the battery status (power), see
https://github.com/Arkapravo/turtlebot/blob/master/turtlebot_bringup/config/diagnostics.yaml#L19